config/webpack.config.js

```diff
@@ -0,0 +1,205 @@
+const path = require('path');
+const webpack = require('webpack');
+const nodeModules = require('webpack-node-externals');
+const StartServerPlugin = require('start-server-webpack-plugin');
+const glob = require('glob');
+
+const NODE_ENV = process.env.NODE_ENV || 'development';
+const assetName = ext => () => NODE_ENV === 'development' ?
+    `assets/[path][name].${ext || '[ext]'}` :
+    `assets/[hash].${ext || '[ext]'}`;
+
+const main = {
+  target: 'node',
+
+  mode: NODE_ENV,
+
+  entry: {
+    main: ['./src/main.js']
+  },
+  output: {
+    path: path.resolve(__dirname, '..', 'dist'),
+    filename: '[name].js',
+    publicPath: '/'
+  },
+
+  devtool: 'source-map',
+
+  externals: ['chokidar'],
+
+  stats: {
+    modules: false,
+    warningsFilter: [
+      // Express uses a dynamic require in view.js but we don't care
+      /node_modules\/express\/lib\/view\.js/
+    ]
+  },
+
+  performance: {
+    // Performance hints not that useful for target:node
+    hints: false
+  },
+
+  module: {
+    rules: [
+      // FIXME: remove when govuk-frontend has support
```

Comment:
I agree and wish that was ok.

But if we want to use the upstream govuk template ... then we really should include everything that is referenced in it because:

1. An update to the govuk_template_jinja module might stop working properly if it changes the js
2. it sets a class on body to alter if in 'js-mode' or not which is used in css
3. the cookie notification would break
4. if we exclude the file, clients would see 404 requests to the missing file since we can't control the script tag that includes it in the template

Reply:
Fair enough!